protocol/group_chat_invitation.go

```diff
@@ -0,0 +1,65 @@
+package protocol
+
+import (
+	"crypto/ecdsa"
+	"encoding/json"
+	"fmt"
+
+	"github.com/golang/protobuf/proto"
+
+	"github.com/status-im/status-go/eth-node/crypto"
+	"github.com/status-im/status-go/eth-node/types"
+	"github.com/status-im/status-go/protocol/protobuf"
+)
+
+// Invitation represents an group chat invitation request from a user in the application layer, used for persistence, querying and
+// signaling
+type GroupChatInvitation struct {
+	protobuf.GroupChatInvitation
+
+	// From is a public key of the author of the invitation.
+	From string `json:"from,omitempty"`
+
+	// SigPubKey is the ecdsa encoded public key of the invitation author
+	SigPubKey *ecdsa.PublicKey `json:"-"`
+
+	// LocalChatID is the chatID of the local chat (one-to-one are not symmetric)
+	LocalChatID string `json:"localChatId"`
+}
```

Comment:
I think that this code was probably copied from the emoji reaction (that change wasn't done by me), but idiomatically tagged struct fields are grouped together at the top and then untagged fields.

```suggestion
type GroupChatInvitation struct {
	protobuf.GroupChatInvitation

	// From is a public key of the author of the invitation.
	From string `json:"from,omitempty"`

	// LocalChatID is the chatID of the local chat (one-to-one are not symmetric)
	LocalChatID string `json:"localChatId"`

	// SigPubKey is the ecdsa encoded public key of the invitation author
	SigPubKey *ecdsa.PublicKey `json:"-"`
}
```